pagopa/pn-frontend
# Feat/pn 2313 gestione informativa privacy e tos

## Short description
Adds Privacy Policy for every application and Terms of service for pn-pa-webapp and pn-personafisica-webapp

## List of changes proposed in this pull request
- Adds Privacy Policy for every application and Terms of service for pn-pa-webapp and pn-personafisica-webapp
- Updates links that bring to privacy policy and tos pages

## How to test
Start pn-personafisica-webapp, go to /privacy-tos and check that both privacy policy and terms of service are visible
Start pn-pa-webapp, go to /privacy-tos and check that both privacy policy and terms of service are visible
Start pn-personafisica-login, go to /informativa-privacy and check that privacy policy is present, while terms of service not
Start pn-landing-webapp, go to /informativa-privacy and check that privacy policy is present, while terms of service not
